challenge_hand_me_that/src/challenge_hand_me_that/get_furniture_from_operator_pose.py

```diff
@@ -0,0 +1,175 @@
+#
+# Copyright (c) 2019, TU/e Robotics, Netherlands
+# All rights reserved.
+#
+# \author Rein Appeldoorn
+
+import os
+
+import rospy
+from geometry_msgs.msg import PoseStamped
+from robot_skills import Hero
+from robot_smach_states import Entity, VariableDesignator
+from robot_smach_states.util.designators import is_writeable
+from rospy import ServiceException
+from smach import StateMachine, cb_interface, CBState
+from std_msgs.msg import Header
+from ed_sensor_integration.srv import RayTraceResponse
+
+
+OPERATOR = None
+all_possible_furniture = ['kitchen_cabinet',
+                          'kitchen_table',
+                          'island',
+                          'sink',
+                          'dishwasher',
+                          'desk',
+                          'coffee_table',
+                          # 'couch',
+                          # 'armchair',
+                          'display_cabinet',
+                          'sideboard']
+
+
+class GetFurnitureFromOperatorPose(StateMachine):
```

Comment:
This state, if cleaned up, could be a nice addition to robot_smach_states.

Reply:
Cleanup is an understatement